pkg/sql/catalog/tabledesc/constraint_test.go

```diff
@@ -19,102 +19,105 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// TestConstraintRetrieval tests the following three method inside catalog.TableDescriptor interface.
-// - AllConstraints() []Constraint
-// - AllActiveAndInactiveConstraints() []Constraint
-// - AllActiveConstraints() []Constraint
+// TestConstraintRetrieval tests the AllConstraints and EnforcedConstraints
+// methods of the catalog.TableDescriptor interface.
 func TestConstraintRetrieval(t *testing.T) {
-	// Construct a table with the following constraints:
-	//  - Primary Index: [ID_1:validated]
-	//  - Indexes: [a non-unique index, ID_4:validated]
-	//  - Checks: [ID_2:validated], [ID_3:unvalidated]
-	//  - OutboundFKs: [ID_6:validated]
-	//  - InboundFKs: [ID_5:validated]
-	//  - UniqueWithoutIndexConstraints: [ID_7:dropping]
-	//  - mutation slice: [ID_7:dropping:UniqueWithoutIndex, ID_8:validating:Check, ID_9:validating:UniqueIndex, a non-unique index]
```

Comment:
I still feel like a comment block like this that summarizes all the constraints we put on this contrived table is helpful.

Reply:
OK I'll add it back.